index.js

```diff
@@ -7,6 +7,9 @@ const gameOverText = document.getElementById("game-over");
 const newGameBtn = document.querySelector(".btn-newGame");
 
 const HIGHSCORESTRING = "highscore";
+const STARTSPEED = 200;
+const SPEEDINCREMENT = 25;
```

Comment:
I think that 25ms is too much. It gets very hard very fast. Could you change it to 5ms?

Reply:
Yup!